src/game/default/g_cmd.c

```diff
@@ -729,6 +754,10 @@ static void G_Vote_f(g_entity_t *ent) {
 
 	ent->client->locals.persistent.vote = VOTE_YES; // client has implicitly voted
 	g_level.votes[VOTE_YES] = 1;
+	
+	if (ent->client->locals.persistent.admin) {
+		g_level.votes[VOTE_YES] = 100;	// admin votes always succeed...
```

Comment:
Or revert their vote count to 1, like everyone else, and add another command for forcing the vote to pass or something? `admin vote pass` | `admin vote fail` Just spitballing..